pkg/quota/admission/clusterresourceoverride/admission.go

```diff
@@ -180,3 +176,39 @@ func (a *clusterResourceOverridePlugin) Admit(attr admission.Attributes) error {
 func multiply(x *inf.Dec, y *inf.Dec) *inf.Dec {
 	return inf.NewDec(0, 0).Mul(x, y)
 }
+
+// wrappedLimitRangerActions wraps the default LimitRangerActions.
+type wrappedLimitRangerActions struct {
+	internalLimitRangerActions limitranger.LimitRangerActions
+}
+
+// wrappedLimitRangerActions implements the LimitRangerActions interface
+var _ limitranger.LimitRangerActions = &wrappedLimitRangerActions{}
+
+// Limit is a pluggable function to enforce limits on the object.  This wraps the default
+// implementation to always return success so that all defaults will be applied.  Validation will
```

Comment:
This was just a refactor due to the new interface introduced upstream, no functionality changes.  This allows  containers to be populated with the default values that can be defined on limits before applying the overrides.  It also goes through the requests on the pod and merges them with the defaults that are applied, annotating the container about changes being made.   Original method:

```
-func wrapLimit(limitRange *kapi.LimitRange, resourceName string, obj runtime.Object) error {
 -	limitranger.Limit(limitRange, resourceName, obj)
 -	// always return success so that all defaults will be applied.
 -	// validation will occur after the overrides.
 -	return nil
 -}
```